src/inlineExpression.tsx

```diff
@@ -85,47 +85,79 @@ export class RenderedExpression extends Widget {
   }
 }
 
-export function InlineRenderer({ value }: { value?: string }): JSX.Element {
-  const ref = useRef<HTMLDivElement>(null);
-  const { cell } = useJupyterCell();
-  // Load the information from the MystMarkdownCell
-  const metadata = getUserExpressions(cell);
-  const trusted = cell?.model.trusted ?? false;
-  // We use the notebook rendermime directly
-  const rendermime = (cell?.parent as StaticNotebook).rendermime;
+function PlainTextRenderer({ content }: { content: string }) {
+  content = content.replace(/^(["'])(.*)\1$/, '$2');
+  return <span>{content}</span>;
```

Comment:
We might want this a bit fancier in the future, e.g. style + alt-text hover with the expression used to create it, but for now this is great!